sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1378,7 +1378,8 @@ public CaseDataDto save(@Valid @NotNull CaseDataDto dto) throws ValidationRuntim
 	@Override
 	@RolesAllowed({
 		UserRight._CASE_CREATE,
-		UserRight._CASE_EDIT })
+		UserRight._CASE_EDIT,
+		UserRight._EXTERNAL_VISITS })
```

Comment:
I would introduce a new method for handling case updates by external visits and should not allow save for external visit right

Reply:
Somehow the method is already for that purpose that @leventegal-she suggested: It's only used by `VisitFacade.saveVisit` but from a variety of origins.

![grafik](https://user-images.githubusercontent.com/24678567/163985211-c1e6235c-3915-463f-8596-8040f6e59054.png)

Reply:
But then why the `save` should be allowed for UserRight._EXTERNAL_VISITS?

Reply:
Looking again on this I think we should maybe exclude the `CaseFacadeEjb.save(CaseDataDto, boolean)` from the `CaseFacade` and change it to private and remove the permissions since it's only internally called by CaseFacadeEjb itself (at least on my mostly updated `development` branch).

Reply:
Discussed with @FredrikSchaeferVitagroup: The state on 1.70.x is different: saveVisit calls directly `CaseFacade.save(caseDataDto, boolean)` and therefore needs that annotation. In #8508 the new method `CaseFacade.updateSymptomsByVisit` was introduced and makes annotations on `CaseFacade.save(caseDataDto, boolean)` superfluous (the usage of that method could be reduced in that ticket or afterwards).